frontend/src/tests/components/Sections/SectionsTable.test.js

```diff
@@ -106,7 +106,7 @@ describe("UserTable tests", () => {
 
         const expectedHeaders = ["Quarter", "Course Number", "Course Title", "Days", "Begin Time", "End Time", 
         "Enrolled", "Max. Enrollment"];
-        const expectedFields = ["courseInfo.quarter", "courseInfo.courseId", "courseInfo.title", "section.timeLocations[0].days", 
+        const expectedFields = ["quarter", "courseInfo.courseId", "courseInfo.title", "section.timeLocations[0].days",                        
```

Comment:
Not sure why there's a ton of spaces here, but we should probably remove it. 

```suggestion
        const expectedFields = ["quarter", "courseInfo.courseId", "courseInfo.title", "section.timeLocations[0].days",
```